src/TurbulenceClosures/abstract_scalar_diffusivity_closure.jl

```diff
@@ -65,7 +65,8 @@ const AVD = AbstractScalarDiffusivity{<:Any, <:VerticalFormulation}
 @inline viscous_flux_vx(i, j, k, grid, closure::AHD, clock, U, args...) = - ν_ζᶠᶠᶜ(i, j, k, grid, clock, closure.ν, U.u, U.v)
 @inline viscous_flux_uy(i, j, k, grid, closure::AHD, clock, U, args...) = + ν_ζᶠᶠᶜ(i, j, k, grid, clock, closure.ν, U.u, U.v)   
 @inline viscous_flux_vy(i, j, k, grid, closure::AHD, clock, U, args...) = - ν_δᶜᶜᶜ(i, j, k, grid, clock, closure.ν, U.u, U.v)
-# are we sure about this?? This might not be consistent for AHD
+
+# TODO: are we sure about this?? This might not be consistent for AHD
```

Comment:
I convinced myself that this is, indeed, consistent in case of AHD for highly anisotropic grids, that is the case in which we would use this formulation...

since the vertical fluxes of horizontal velocities do not have horizontal gradients of the vertical velocity, the horizontal flux of vertical velocity should not contain vertical gradients of horizontal velocity...

Reply:
I think I agree, but what do you mean that it's dependent on grid anisotropy? There's no check for grid anisotropy here?

Reply:
It feels like the issue is that the formulation of some terms really depends on whether or not the model is hydrostatic or not. Coriolis forces are the same; if we are non-hydrostatic we include some terms that we wouldn't otherwise. I think we might need a type to describe whether a model is hydrostatic (or we can use a numeric multiplier like MITgcm)

Reply:
No there is not, but when the grid is very anisotropic it is better to use the `Horizontal` formulation that the `ThreeDimensional` formulation.

talking with @jm-c we came to the conclusion that is not desired to have the horizontal flux of vertical velocity dominated by the vertical gradients of horizontal velocity (which might be very large!) which would be the case in terms of a very anisotropic grid.

Reply:
But we don't have any information about grid anisotropy here?

Reply:
I'm not sure I agree with the argument.

What we want is an interface that allows users to define the PDE they want to solve numerically. We should definitely avoid psychology / assumptions about what users are trying to do (eg "if users select `HorizontalScalarDiffusivity` then _of course_ they are using an anisotropic grid... right?!?)

If there are two legitimate choices then we need a way to express the difference between the two in our API...

Reply:
Since we've already appended our types with the word "formulation" we can maybe leverage that to add a "horizontal diffusion" that would be appropriate for nearly isotropic grids, right? It's not a high priority...

Reply:
> If the grid is quite isotropic (Lx ≈ Ly ≈ Lz) probably it is better to use the ThreeDimensional formulation as maybe the Horizontal formulation removes an important part of the dissipation. I was maybe worried about that... (probably we should write it somewhere in the docs)

What if we really do want diffusion only in the horizontal, but with an isotropic grid? That is the question.